trustyai/explainers.py

```diff
@@ -403,16 +407,17 @@ def as_dataframe(self) -> pd.DataFrame:
 
         Returns
         -------
-        pandas.DataFrame
-            DataFrame containing the results of the SHAP explanation. For each model output,
+        Dict[String, pandas.DataFrame]
+            Dictionary of dataFrame containing the results of the SHAP explanation, keyed by the
+             model output named. For each model output,
```

Comment:
Is this change related? It seems a leftover from a different branch/pr

Reply:
unrelated, must have snuck in